servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -568,6 +575,13 @@ float LightStorage::reflection_probe_get_intensity(RID p_probe) const {
 	return reflection_probe->intensity;
 }
 
+float LightStorage::reflection_probe_get_fade_start(RID p_probe) const {
+	const ReflectionProbe *reflection_probe = reflection_probe_owner.get_or_null(p_probe);
+	ERR_FAIL_COND_V(!reflection_probe, 0);
```

Comment:
```suggestion
	ERR_FAIL_NULL_V(reflection_probe, 0);
```

Reply:
Note that LightStorage is using `ERR_FAIL_COND_V(!reflection_probe` all over the place already, so it may be better to fix that in a dedicated PR.

Reply:
They will, but I'd like to not have to add these as well in my PR

Reply:
Or rather my pr has been approved for this part so I'd have to do a separate one after in that case

Reply:
And merged :)